pennylane/devices/experimental/device_api.py

```diff
@@ -168,6 +178,11 @@ def shots(self) -> Shots:
         """
         return self._shots
 
+    @property
+    def wires(self) -> Wires:
+        """The device wires. None means any wires can be used."""
```

Comment:
We should flesh this docstring out a bit more.

I think we should also add a block to the class docstring for `Required Keyword Arguments:` and explain the role of shots and wires there.  We need to fully spec out what we want to allow for device wires and how it has to affect the computation.

Reply:
I updated this docstring a bit more. I'm good to improve the class docstring but perhaps we can do that in another PR?